common/djangoapps/student/tests/test_views.py

```diff
@@ -154,10 +154,13 @@ def test_course_run_refund_status_successful(self):
 
     def test_course_run_refund_status_invalid_course_key(self):
         """ Assert that view:course_run_refund_status returns correct Json for Invalid Course Key ."""
-        with patch('opaque_keys.edx.keys.CourseKey.from_string') as mock_method:
-            mock_method.side_effect = InvalidKeyError('CourseKey', 'The course key used to get refund status caused \
-                                                        InvalidKeyError during look up.')
-            response = self.client.get(reverse('course_run_refund_status', kwargs={'course_id': self.course.id}))
+        test_url = reverse('course_run_refund_status', kwargs={'course_id': self.course.id})
```

Comment:
Note: This test did not seem related to my changes, but the reverse was failing with an invalid course key exception in its old location due to the mock. The test goal seems to test the View method and not reverse, so I moved the check up. With that change the view code is now executed and the test is passing.